### PR TITLE
Add local comparison tool

### PR DESCRIPTION
**Pull request by lxtbell**
Supports comparing several versions of local annotation JSON files, showing only different parts.
Note that changes under public/brat_client are for bumping the brat client version.

### CODE REVIEW

File: comparison.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+var argv = minimist(process.argv.slice(2));
+var serverPort = argv['port'] || 8080;
+var annotationFolders = argv['_'];
+if (annotationFolders.length == 0) {
+    annotationFolders = [__dirname + '/public/comparison/prediction', __dirname + '/public/comparison/gold'];
+}
+
+
+// Retrieve file list for every request might be slow, while allowing adding new files without restarting the server
```
Comment (mssammon) [inline comparison.js]: As written, this comment is ambiguous: what decision was made here OR how does this method support a choice by the user/client?

**Comment by danyaljj** [comment]
With respect to some of the third part libraries added: can you add them via npm? This you don't need to copy / paste the library itself in the codebase.

**Comment by danyaljj** [comment]
@lxtbell let me know if you need any help on this.

**Comment by lxtbell** [comment]
Now rebased onto #24.

**Pull request merged**